src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/notification/ClaimSubmissionLipClaimantNotificationHandler.java

```diff
@@ -0,0 +1,73 @@
+package uk.gov.hmcts.reform.civil.handler.callback.camunda.notification;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.ccd.client.model.AboutToStartOrSubmitCallbackResponse;
+import uk.gov.hmcts.reform.ccd.client.model.CallbackResponse;
+import uk.gov.hmcts.reform.civil.callback.Callback;
+import uk.gov.hmcts.reform.civil.callback.CallbackHandler;
+import uk.gov.hmcts.reform.civil.callback.CallbackParams;
+import uk.gov.hmcts.reform.civil.callback.CaseEvent;
+import uk.gov.hmcts.reform.civil.model.CaseData;
+import uk.gov.hmcts.reform.civil.notify.NotificationService;
+import uk.gov.hmcts.reform.civil.notify.NotificationsProperties;
+
+import java.util.List;
+import java.util.Map;
+
+import static uk.gov.hmcts.reform.civil.callback.CallbackType.ABOUT_TO_SUBMIT;
+import static uk.gov.hmcts.reform.civil.callback.CaseEvent.NOTIFY_LIP_CLAIMANT_CLAIM_SUBMISSION;
+
+@Service
+@RequiredArgsConstructor
+public class ClaimSubmissionLipClaimantNotificationHandler extends CallbackHandler implements NotificationData {
+
+    private static final List<CaseEvent> EVENTS = List.of(
+        NOTIFY_LIP_CLAIMANT_CLAIM_SUBMISSION);
+    public static final String TASK_ID = "ClaimSubmissionLipClaimantNotification";
+    private static final String REFERENCE_TEMPLATE =
+        "claim-submission-lip-claimant-notification-%s";
+
+    private final NotificationService notificationService;
+    private final NotificationsProperties notificationsProperties;
+
+    @Override
+    protected Map<String, Callback> callbacks() {
+        return Map.of(
+            callbackKey(ABOUT_TO_SUBMIT),
+            this::notifyLipClaimantForClaimSubmission
+        );
+    }
+
+    @Override
+    public String camundaActivityId(CallbackParams callbackParams) {
+        return TASK_ID;
+    }
+
+    @Override
+    public List<CaseEvent> handledEvents() {
+        return EVENTS;
+    }
+
+    private CallbackResponse notifyLipClaimantForClaimSubmission(
+        CallbackParams callbackParams) {
+        CaseData caseData = callbackParams.getCaseData();
+        notificationService.sendMail(
```

Comment:
better to add a validation for email before sendMail call.

Reply:
Updated as suggested